README.md

```diff
@@ -0,0 +1,73 @@
+# [ipv6.watch](https://ipv6.watch)
+
+[![https://img.shields.io/travis/andir/ipv6.watch.svg](https://travis-ci.org/andir/ipv6.watch)]
+
+Although IPv6 has been around since 1998, some big parts of the internet are IPv4 only. Since World IPv6 Launch day on June 6 2012 the global IPv6 traffic has grown more than 500%. However, the adoption at major websites is still bad even though they should have the money, time and people to provide IPv6 connectivity.
+
+This repository contains code to check some website domains for AAAA records using multipe public available DNS resolvers and generate a static website containing the results.
+
+![](https://raw.githubusercontent.com/andir/ipv6.watch/master/misc/World_IPv6_launch_banner_512.png)
+
+## Contributing + Dev info
+
+### Missing websites or website infos
+
+You can report missing websites or domains by opening an issue. If you want to provide them yourself have a look at the developer documentation down below.
```

Comment:
While I never intended to offer a service to add websites that paragraph is probably fine. There seem to be enough people interested in this project to distribute the load.